backend/btrixcloud/models.py

```diff
@@ -506,7 +506,7 @@ class CollIn(BaseModel):
     description: Optional[str]
     crawlIds: Optional[List[str]] = []
 
-    isPublic: Optional[bool] = False
```

Comment:
What's the reason for this change? Perhaps should just keep as is to be consistent?

Reply:
I'm following what the frontend sends, so I had to change the model. I attached screenshots in the comments about how this is inconsistent, but this currently accurate represents the data of the request.